### PR TITLE
remove unused license code

### PR DESCRIPTION
**Pull request by z7658329**

### CODE REVIEW

**Review by zyxxoo — APPROVED** [review]

**Comment by imbajin** [comment]
also remove  the whole license header in swagger files (if it's imported)

image

File: hugegraph-api/src/main/java/org/apache/hugegraph/api/API.java [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /*
- * Copyright 2017 HugeGraph Authors
```
Comment (javeme) [inline hugegraph-api/src/main/java/org/apache/hugegraph/api/API.java:2]: can also remove the next blank line?

**Review by javeme — APPROVED** [review]

File: hugegraph-api/src/main/java/org/apache/hugegraph/api/API.java (hunk 1 above)
Comment (imbajin) [reply hugegraph-api/src/main/java/org/apache/hugegraph/api/API.java:2]: @z7658329 I remove the extra line in commons & toolchains, you could update the model for server

and should check other files, like `Copyright 2019`, the regex settings shall be carefully

Comment (z7658329) [reply hugegraph-api/src/main/java/org/apache/hugegraph/api/API.java:2]: > @z7658329 I remove the extra line in [commons](https://github.com/apache/incubator-hugegraph-commons/pull/124/commits/edc56b8e5f7b37e9819cd8a24b07ac1fa8e4e23c) & [toolchains](https://github.com/apache/incubator-hugegraph-toolchain/pull/421), you could update the model for server
> 
> and should check other files, like `Copyright 2019`, the regex settings shall be carefully

ok  i will check it

**Comment by z7658329** [comment]
see #2090

**Pull request closed without merging**